sklearn/tests/metadata_routing_common.py

```diff
@@ -259,13 +259,10 @@ def predict(self, X, sample_weight="default", metadata="default"):
         return np.zeros(shape=(len(X),))
 
     def predict_proba(self, X, sample_weight="default", metadata="default"):
-        pass  # pragma: no cover
-
-        # uncomment when needed
-        # record_metadata_not_default(
-        #     self, "predict_proba", sample_weight=sample_weight, metadata=metadata
-        # )
-        # return np.asarray([[0.0, 1.0]] * len(X))
+        record_metadata_not_default(
+            self, "predict_proba", sample_weight=sample_weight, metadata=metadata
+        )
+        return np.asarray([[0.0, 1.0]] * len(X))
```

Comment:
where is this used?

Reply:
In `StackingClassifier`, there's a `stack_method` param, whiches value can be set to `predict_proba` as well. In StackingRregessor it's just hardcoded as `predict`.